x-pack/test_serverless/api_integration/services/transform/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+import { TransformAPIProvider } from './api';
+import { TransformSecurityCommonProvider } from './security_common';
+
+export function TransformProvider(context: FtrProviderContext) {
+  const api = TransformAPIProvider(context);
+  const securityCommon = TransformSecurityCommonProvider(context);
+
+  return {
+    api,
```

Comment:
I moved the transform service from stateful to serverless. I already make use of some api methods. `securityCommon` will be used later on for SLO permissions tests